src/components/common/Dialogs/ControlSafeDialog/AmountBalances/AmountBalances.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect } from 'react';
+import { defineMessages } from 'react-intl';
+import classnames from 'classnames';
+import { useFormContext } from 'react-hook-form';
+
+import { HookFormInput as Input, TokenSymbolSelector } from '~shared/Fields';
+import {
+  ADDRESS_ZERO,
+  /* BINANCE_NETWORK, */
+  DEFAULT_TOKEN_DECIMALS,
+  ETHEREUM_NETWORK,
+  SAFE_NETWORKS,
+} from '~constants';
+import { MaxButtonParams } from '~shared/Fields/Input/HookForm';
+import { Safe, SafeBalance, Token } from '~types';
+import { TokenType } from '~gql';
+
+import styles from './AmountTokens.css';
+/* import { TransferFundsProps } from '../TransactionTypesSection/TransferFundsSection'; */
+
+/* interface Props extends Pick<TransferFundsProps, 'handleValidation'> { */
+interface Props {
+  safeBalances: SafeBalance[];
+  disabledInput: boolean;
+  selectedSafe: Safe | undefined;
+  transactionFormIndex: number;
+  maxButtonParams: MaxButtonParams;
+  /* handleChange: () => void; */
+}
+
+const displayName = 'common.ControlSafeDialog.AmountBalances';
```

Comment:
I think this AmountBalances is used only inside `ControlSafeForm`. Therefore the id should be `common.ControlSafeDialog.ControlSafeForm.AmountBalances`